### ISSUE TITLE
Raiffeisen Smart Mobile

### ISSUE THREAD

**Issue by wetbandits6** [PrivSec-dev/banking-apps-compat-report#86]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### App name

Raiffeisen Smart Mobile

### Link to app

https://play.google.com/store/apps/details?id=ro.raiffeisen.smartmobile

### App version

4.26.1.1

### Country of the app

Romania

### Build Number

SQ3A.220605.009.A1.2022063000

### Device

- [ ] Pixel 6 Pro
- [ ] Pixel 6
- [ ] Pixel 5a
- [ ] Pixel 5
- [ ] Pixel 4a (5G)
- [ ] Pixel 4a
- [ ] Pixel 4 XL
- [ ] Pixel 4
- [ ] Pixel 3a XL
- [ ] Pixel 3a
- [ ] Pixel 3 XL
- [ ] Pixel 3

### Google Play installed (tried both)

- [X] Installed
- [X] Not installed

### Google Play services Network permission revoked?

- [ ] Revoked
- [X] Not revoked
- [ ] I did not have Google Play services installed

### SafetyNet Enforcement

- [ ] Enforced
- [X] Not enforced
- [ ] Unsure

### Native code debugging

- [ ] Enabled
- [X] Disabled

### Stock OS compatibility

- [X] Works (with or without Graphene OS sandboxed Google Play services)
- [ ] Does not work
- [ ] Not tested

### Profile app tested in

- [ ] Main user profile
- [ ] Secondary user profile
- [ ] I used a work profile device manager app (Please mention more details about it down below like the app)

### Description of the app's functionality

Everything seems to work. It sometimes crashes when used without Google Play Services.

### Are there any extra notes you think users should know about?

The companion app, Raiffeisen SmartToken also works.

**Comment by akc3n** [comment]
Hi @wetbandits6 thanks for submitting this. May you provide the device and the type of profile used in this report please. However, if you prefer not to due to your opsec/threat model, then that's all good too.

**Comment by oancea-cristian** [comment]
I've noticed today that Raiffeisen does not work on my Google Pixel 8 anymore, when I open the app it opens a web page instead telling me I can't use the app on a rooted device. It worked fine since I installed Graphene OS about 1 year ago but now it's not anymore.

**Comment by tudorbobei** [comment]
I've encountered the same issue on a Pixel 9. When I launch the app, it redirects to a web page stating that I can't use the app on a rooted device. However, there is no notification about Google API hardware attestation, which suggests the app is detecting a modified operating system through some other method.

I've tested with Exploit Protection compatibility mode enabled and sandboxed Google Play services with network permissions allowed, but the issue still persists. The app was working fine until just until five days ago.

**Comment by spring-onion** [comment]
@oancea-cristian @dwrtewd Do you get a notification that it's using the play integrity api?

**Comment by oancea-cristian** [comment]
No, I don't but I've changed that notifications to not show a long time ago and I am not sure how to turn in back on. I never allowed access and it worked fine.

**Comment by spring-onion** [comment]
Sorry I missed you mentioned that already. My testing matches your experience, I tried disabling all exploit protection settings (including secure app spawning) to no avail. Please refer them to the [attestation compatibility guide](https://grapheneos.org/articles/attestation-compatibility-guide).

**Comment by oancea-cristian** [comment]
I've just tested the application and is now running again as expected. The problem seems to have been fixed on their end (I didn't do anything).
